packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -205,11 +205,19 @@ export default function() {
       },
 
       TSImportEqualsDeclaration(path) {
-        throw path.buildCodeFrameError("`import =` is not supported.");
+        throw path.buildCodeFrameError(
+          "`import =` is not supported in Babel's Typescript parser " +
+            "because there is no valid ES6 implementation for it.\nPlease consider using " +
+            "`import <moduleName> from '<moduleName>';` alongside " +
+            "Typescript's --allowSyntheticDefaultImports option.",
+        );
       },
 
       TSExportAssignment(path) {
-        throw path.buildCodeFrameError("`export =` is not supported.");
+        throw path.buildCodeFrameError(
+          "`export =` is not supported in Babel's Typescript parser because " +
+            "there is no valid ES6 implementation for it.\nPlease consider using `export <value>;`.",
```

Comment:
Isn't `export =` more like `export default <value>` than `export <value>` (I don't use TS very much, so I'm not sure)

Reply:
Yes.  Basically you could only export one object earlier. Now support 
```export default <value>``` & ```export <value>```